backend/src/services/email/email-template.service.ts

```
@@ -82,7 +84,7 @@ export class EmailTemplateService {
    * Get a single email template by type
    * Throws 404 if not found
    */
-  async getTemplate(templateType: string): Promise<EmailTemplateSchema> {
+  async getTemplate(templateType: string, providerType: string = 'custom_smtp'): Promise<EmailTemplateSchema> {
```

ESLint

[error] Replace `templateType:·string,·providerType:·string·=·'custom_smtp'` with `⏎····templateType:·string,⏎····providerType:·string·=·'custom_smtp'⏎··`

(prettier/prettier)